ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -302,6 +302,10 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure
+- as: e2e-powervs
+  cron: 0 0 * * 1
+  steps:
+    workflow: openshift-e2e-powervs-upi
```

Comment:
@rpsene add this content above the `workflow` line to change the tests that run here to a simple `oc get nodes` instead of the full suite:

```yaml
    test:
    - as: test-e2e
      cli: latest
      commands: |
        oc get nodes
      from: base
      resources:
        requests:
          cpu: 100m
```